### ISSUE TITLE
Unable to Export Ticket List

### ISSUE THREAD

**Issue by mdkaleel** [Project-Pier/ProjectPier-Core#11]
Hi,

Unable to Export Ticket Please Help,

```php
Session "default" started at 2015-01-05T07:44:53+0000 
#1 ERROR: Error: Array to string conversion in 'E:\PPNew\application\models\application_logs\ApplicationLogs.class.php' on line 154 (error code: 8) 
#2 ERROR: Error: Array to string conversion in 'E:\PPNew\application\models\application_logs\ApplicationLogs.class.php' on line 160 (error code: 8) 
#3 ERROR: Error: Array to string conversion in 'E:\PPNew\application\models\application_logs\ApplicationLogs.class.php' on line 168 (error code: 8) 
#4 ERROR: Error: Array to string conversion in 'E:\PPNew\application\models\application_logs\ApplicationLogs.class.php' on line 160 (error code: 8) 
#5 ERROR: Error: Array to string conversion in 'E:\PPNew\application\models\application_logs\ApplicationLogs.class.php' on line 168 (error code: 8) 
#6 ERROR: Error: Array to string conversion in 'E:\PPNew\application\models\application_logs\ApplicationLogs.class.php' on line 160 (error code: 8) 
#7 ERROR: Error: Array to string conversion in 'E:\PPNew\application\models\application_logs\ApplicationLogs.class.php' on line 168 (error code: 8) 
#8 ERROR: Error: Array to string conversion in 'E:\PPNew\application\models\application_logs\ApplicationLogs.class.php' on line 154 (error code: 8) 
#9 ERROR: Error: Array to string conversion in 'E:\PPNew\application\models\application_logs\ApplicationLogs.class.php' on line 160 (error code: 8) 
#10 ERROR: Error: Array to string conversion in 'E:\PPNew\application\models\application_logs\ApplicationLogs.class.php' on line 168 (error code: 8) 
#11 ERROR: Error: Array to string conversion in 'E:\PPNew\application\models\application_logs\ApplicationLogs.class.php' on line 160 (error code: 8) 
#12 ERROR: Error: Array to string conversion in 'E:\PPNew\application\models\application_logs\ApplicationLogs.class.php' on line 168 (error code: 8) 
#13 ERROR: Error: Array to string conversion in 'E:\PPNew\application\models\application_logs\ApplicationLogs.class.php' on line 160 (error code: 8) 
#14 ERROR: Error: Array to string conversion in 'E:\PPNew\application\models\application_logs\ApplicationLogs.class.php' on line 168 (error code: 8) 
Time since start: 0.51563215255737 seconds 
```

**Comment by JonDeG** [comment]
What version of the application are you using and what version of PHP are you using?

**Comment by Engelbert** [comment]
On a side note: With a fresh install from ProjectPier-Core, on a local server that is compatible according to the docs, I do get an empty .txt file when I download the tickets list. This probably is a bug. Can someone confirm this?

**Comment by JonDeG** [comment]
I can confirm, the Ticket list always produces an empty text file on the latest PP version from GitHub running on PHP 5.3.

**Comment by mdkaleel** [comment]
PHP Version :5.4.14
ProjectPier Version : 0.8.8.sp2

**Comment by Engelbert** [comment]
This issue is no longer needed. 
### It will be fixed as a part of issue #17 
[CLOSED]
